docs/gambit/gambit.md

```diff
@@ -242,12 +242,11 @@ Here are some examples of using the `--sourceroot` option.
    ```
 
    This should output the following:
-   <!-- Code output: using `pre` to avoid the Copy To Clipboard feature -->
-   <pre>
+   ```
```

Comment:
I'm fine with this change, although IIRC this code is automatically generated; is this change in sync with changes in the original source?

Reply:
You are right that we have a PR for auto generated doc, but that PR is on hold for a bit. When that PR lands, it is expected to handle this correctly (i.e., change it to ``` when needed). Here, this was just a copy paste error.